components/message/index.tsx

```diff
@@ -29,7 +29,7 @@ type NoticeType = 'info' | 'success' | 'error' | 'warning' | 'loading';
 
 function notice(
   content: React.ReactNode,
-  duration: number = defaultDuration,
+  duration: (() => void) | number = defaultDuration,
```

Comment:
Supply multiple function types for the same function as a list of overloads will be better.

Reply:
Thanks for your review, you mean ?

```js
duration: (() => void) | number = defaultDuration,
```

to 

```js
const fn = () => void;
...

duration: fn | number = defaultDuration,
```

Reply:
> Supply multiple function types for the same function as a list of overloads will be better.  

why is it better ?

Reply:
Sorry, I got it wrong, you are right.

Reply:
No problem, just asking ;)